rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml

```diff
@@ -24,13 +24,14 @@
     - logstash-web-disable
     - logstash-post-install
 
-- name: Stop logstash-web
-  service:
-    name: logstash-web
-    state: stopped
-    enabled: no
+- name: Create patterns directory
+  file:
+    name: "/opt/logstash/patterns"
+    owner: "logstash"
+    group: "logstash"
+    state: directory
   tags:
-    - logstash-web-disable
```

Comment:
That's actually because the `Create patterns directory` is in the same place and is the same number of lines as the defunct and removed `Logstash web disable` task.